executor/src/witgen/jit/effect.rs

```diff
@@ -60,6 +60,27 @@ pub struct Assertion<T: FieldElement, V> {
     pub expected_equal: bool,
 }
 
+impl<T: FieldElement> Effect<T, Variable> {
```

Comment:
This way it is between `struct Assertion` and its impl. Can you move it up to effect?